### PR TITLE
Fixed broken pics and re-ordered the document.

### PR DESCRIPTION
**Pull request by ArikDay**
## Status
Ready

## Description
Fixed broken pics and re-ordered the document.

### CODE REVIEW

File: content-repo/extra-docs/packs/Email-Communication-Readme.md [hunk 1]
```diff
@@ -94,36 +94,6 @@ In this layout there are three interactive sections in which you can specify one
 
 ---
 
-#### Email Threads Layout
```
[the remark this answers is not in the document]
Comment (ArikDay) [reply content-repo/extra-docs/packs/Email-Communication-Readme.md:97]: Yes, that's a good idea.
this way the its more suitable to "In this Pack" and the second part is configurations and instructions.
Will push the changes.

[the remark this answers is not in the document]
Comment (ArikDay) [reply content-repo/extra-docs/packs/Email-Communication-Readme.md:97]: also discovered that the hyperlink of "Configure the service_mail and mail_sender_instance mailsenderinstance-parameters" is broken. will fix it also

File: content-repo/extra-docs/packs/Email-Communication-Readme.md [hunk 2]
```diff
@@ -70,58 +70,6 @@ The Email Communication content pack includes several content items.
 * **Email Communication**: Used in the **Email Communication** incident type.
 * **Email Threads**: This layout can be added to other incident types to add email functionality to them.
```
Comment (altmannyarden) [inline content-repo/extra-docs/packs/Email-Communication-Readme.md:71]: Maybe let's add a link to their section (for example for the email communication - add hyperlink to the Layout Instructions  -> Email Communication Layout

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.  Fixed one broken cross-ref.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**